packages/domains-table/src/domains-table/domains-table-row.tsx

```diff
@@ -197,6 +200,17 @@ export function DomainsTableRow( { domain }: DomainsTableRowProps ) {
 					);
 				}
 
+				if ( column.name === 'ssl' && config.isEnabled( 'hosting-overview-refinements' ) ) {
+					return (
+						<td key={ domain.domain + column.name }>
+							<DomainsTableSSLCell
+								domainManagementLink={ domainManagementLink }
+								sslStatus={ sslStatus }
+							/>
+						</td>
```

Comment:
```suggestion
						<DomainsTableSSLCell
							key={ domain.domain + column.name }
							domainManagementLink={ domainManagementLink }
							sslStatus={ sslStatus }
						/>
```
Let's move the `<td>` inside `DomainsTableSSLCell` and drop the `div`. That'll make styling easier.